packages/rum-core/src/domain/resource/resourceUtils.ts

```diff
@@ -163,8 +161,10 @@ export function toValidEntry(entry: RumPerformanceResourceTiming) {
     : true
 
   if (areCommonTimingsInOrder && areRedirectionTimingsInOrder) {
-    return entry
+    return true
   }
+
+  return false
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏this statement could be simplified
```
if (x) {
  return true     =>    return x
}
return false
```

Reply:
I'm not sure I understand, do you mean:
```diff
  if (areCommonTimingsInOrder && areRedirectionTimingsInOrder) {
-    return true
+    return areCommonTimingsInOrder && areRedirectionTimingsInOrder
  }

  return false
```

I'm not convinced it's a simplification, but I can make the change anyway. Or maybe I didn't understood your suggestion.

Reply:
I meant:
```diff
-  if (areCommonTimingsInOrder && areRedirectionTimingsInOrder) {
-    return true  
-  }
-
-  return false
+ return areCommonTimingsInOrder && areRedirectionTimingsInOrder
```

Reply:
🤦 of course!